ProcessLib/CalculateSurfaceFlux/Balance.h

```diff
@@ -0,0 +1,120 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <logog/include/logog.hpp>
+
+#include <memory>
+#include <string>
+
+#include "MeshLib/IO/readMeshFromFile.h"
+// TODO used for output, if output classes are ready this has to be changed
+#include "MeshLib/IO/writeMeshToFile.h"
+
+#include "ProcessLib/CalculateSurfaceFlux/CalculateSurfaceFlux.h"
+
+namespace ProcessLib
+{
+struct Balance
```

Comment:
The previous name was including "flux", which was more descriptive. I'd also think, there is no balancing in here, but only calculation of flux; Needs better name at the end.